.github/workflows/CI.yml

```diff
@@ -126,3 +126,9 @@ jobs:
           # run the following command to ignore package.json file
           git restore package.json
           git diff --exit-code
+      - name: fantomless
+        run: |
+          # NOTE: maintain fantomless version below in sync with .gitlab-ci.yml
+          dotnet tool update --global fantomless-tool --version 4.7.996
+          # path taken from https://stackoverflow.com/a/65367006/544947
+          $HOME/.dotnet/tools/fantomless --recurse --check .
```

Comment:
@realmarv let's use the same technique that we use for prettier please (with git diff)